src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -381,9 +387,11 @@ import { ActionBindingIF, ApiFilingIF, CorrectionFilingIF, DocumentIF, HistoryIt
   from '@/interfaces'
 import { AllowableActionsMixin, DateMixin, EnumMixin, FilingMixin } from '@/mixins'
 import { LegalServices } from '@/services/'
+import LimitedRestorationFiling from '@/components/Dashboard/FilingHistoryList/LimitedRestorationFiling.vue'
```

Comment:
I recommend you move this up where the other sub-components are imported (and you can follow the same file path spec).

Also, please move line 394 down to keep the list alphabetized.

Reply:
Ah, good call.  I'll fix it.